### DISCUSSION TITLE
Request to add additional information through API

### DISCUSSION THREAD

**Discussion by Eugeniusz86** [allegro/allegro-api#10451] [Suggestions]
I would like to request an access to the API interface in order to integrate it into our workflow. 
To communicate effectively through the API, we need access to the following data of some existing auctions from different sellers: 

• Product availability (quantity listed on Allegro)
• Auction ID (auction number)

i believe this info is available publicly and can be reached via web interface
thank you
Yevhen

**Comment by MaciejFrackowiak** [comment]
I moved your case to the thread https://github.com/allegro/allegro-api/issues/10454 in the Issues tab.
